internal/proto/controller/storage/servers/store/v1/root_certificate.proto

```diff
@@ -37,10 +37,15 @@ message RootCertificate {
   // @inject_tag: `gorm:"primary_key"`
   bytes public_key = 50;
 
+  // The plain-text private key associated with this certificate. We are not storing this
+  // in the database.
+  // @inject_tag: `gorm:"-"` wrapping:"pt,private_key"`
+  bytes private_key = 60;
+
   // The private key associated with this certificate
   // This is a ciphertext field
-  // @inject_tag: `gorm:"not_null"`
-  bytes private_key = 60;
+  // @inject_tag: `gorm:"column:private_key;not_null"` wrapping:"ct,private_key"`
+  bytes ct_private_key = 65;
```

Comment:
Nit: Just make it 61, I think. AFAIK the splitting of these numbers is to allow us to add more related fields, and this is very very related to the above.